src/core/main.c

```diff
@@ -821,6 +821,7 @@ static int parse_argv(int argc, char *argv[]) {
         };
 
         int c, r;
+        bool arg_user_seen = false;
```

Comment:
`arg_` prefix is not suitable for local variable?

Reply:
Indeed. I renamed it to `user_arg_seen`.

Reply:
call it "dash_dash_user_seen"? :-)

Reply:
Well, yeah, maybe. My version is shorter therefore better!